projects/dynamicforms/src/core/resolvers/utils/forms-utils.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { FormsUtils } from './forms-utils';
+
+describe('FormsUtils', () => {
+  it('should converte to date', () => {
+    const date = '29-01-1990';
+    expect(FormsUtils.convertIfDate(date)).toEqual('1990-01-29');
+  });
+
+  it('should get value from object by path', () => {
+    const fieldPath =
+      'insuranceQuote.insuredObjectList.insuredObjects[0].childInsuredObjectList.insuredObjects[0].dateOfBirth';
```

Comment:
As this is a unit test, there is no need to invoke a real use-case. We need to test getting value from any object so this fieldPath can be 'testObject.listAttribute[0].testAttribute